internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceTwoLayerMultiRegionMultiClusterWrapper.java

```diff
@@ -448,4 +452,28 @@ public void logMultiCluster() {
       }
     }
   }
+
+  public void useLeaderErrorNotifier(String regions) {
```

Comment:
Cool!! Thanks for adding this. nit: can rename it to something like: `failPushInRegion()`